x-pack/filebeat/input/entityanalytics/provider/okta/okta.go

```diff
@@ -465,3 +615,32 @@ func (p *oktaInput) publishUser(u *User, state *stateStore, inputID string, clie
 
 	client.Publish(event)
 }
+
+// publishDevice will publish a device document using the given beat.Client.
+func (p *oktaInput) publishDevice(d *Device, state *stateStore, inputID string, client beat.Client, tracker *kvstore.TxTracker) {
+	devDoc := mapstr.M{}
+
+	_, _ = devDoc.Put("okta", d.Device)
```

Comment:
Does the .Put method return an error, if yes, is it ok to ignore here ?

Reply:
Yes, this is fine to ignore. The errors that `mapstr.M.Put` returns are based on path traversal failures. There is no path to traverse here.